app/src/main/java/io/intrepid/animationexercise/screens/detail/DetailActivity.java

```diff
@@ -101,6 +118,16 @@ public void showCat() {
     public void animatePowerUp() {
         Drawable drawable = powerUpBar.getDrawable();
         drawable.setLevel(10000);
+        ObjectAnimator moveAnim = ObjectAnimator.ofFloat(powerUpBar, "Y", 1000);
+        moveAnim.setDuration(20000);
```

Comment:
20000 equals 20 seconds I believe, which is a little too long. We were looking for something around half a second or 500 ms.